feature/search/src/main/kotlin/uk/govuk/app/search/SearchViewModel.kt

```diff
@@ -58,8 +60,9 @@ internal class SearchViewModel @Inject constructor(
         analytics.search(searchTerm)
     }
 
-    fun onSearchResultClicked(title: String, url: String) {
+    suspend fun onSearchResultClicked(title: String, url: String) {
```

Comment:
This doesn't need to be a suspend function, it can launch a coroutine via the view model scope.

Reply:
Fixed up